satpy/readers/mviri_l1b_fiduceo_nc.py

```diff
@@ -513,11 +534,17 @@ def get_time(self):
         """Get time coordinate.
 
         Variable is sometimes named "time" and sometimes "time_ir_wv".
+        FillValues in time are set to NaT, others converted to datetime64.
         """
         try:
-            return self["time_ir_wv"]
+            time = self["time_ir_wv"]
         except KeyError:
-            return self["time"]
+            time = self["time"]
+
+        time = xr.where(time == time.attrs["_FillValue"], np.datetime64("NaT"),
+                        (time + time.attrs["add_offset"]).astype("datetime64[s]").astype("datetime64[ns]"))
```

Comment:
Is it possible to do this once after decoding?

Reply:
changed according to the results of our discussion last week.